content/en/docs/languages/java/configuration.md

```diff
@@ -424,7 +424,7 @@ of the file alone dictate SDK configuration. {{% /alert %}}
 
 File configuration allows for configuration via a YAML as described in
 [opentelemetry-configuration](https://github.com/open-telemetry/opentelemetry-configuration)
-and [file configuration](/docs/specs/otel/configuration/file-configuration/).
+and [declarative configuration](http://localhost:1313/docs/specs/otel/configuration/#declarative-configuration).
```

Comment:
@jack-berg this is only a minimal change, you might consider reviewing that section in #4966 as well